File: code3.cpp
//extremes of the array
#include <iostream>
using namespace std;

int main(){

  int n ; 
  cout << "Enter the number of elements in the array: ";
  cin >> n;
  int arr[n];
  cout << "Enter the elements of the array: ";
  for ( auto &i : arr){
    cin >> i ;
  }

  int left = 0;
  int right = n-1;
      while( left < right){

      cout << arr[left] << " " << arr[right] << endl;
      left++;
      right--;
      if(left == right){
        cout << arr[left] << endl;
      }
    }

  return 0;

}

File: code3.java
import java.util.Scanner;

public class code3 {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    System.out.println("Enter the size of the array:");
    int n = sc.nextInt();
    int arr[] = new int[n];
    for (int i = 0; i < arr.length; i++) {
      arr[i] = sc.nextInt();
    }

    System.out.println("Extremes values in the array are:");
    int left = 0;
    int right = n-1;
    while (left < right){
      System.out.println(arr[left] + " " + arr[right]);
      left++;
      right--;
      if( left == right){
        System.out.println(arr[left]);
      }
    }
    sc.close();



  }

}
